lib/avromatic/model/configurable.rb

```diff
@@ -8,14 +8,19 @@ module Configurable
 
       module ClassMethods
         attr_accessor :config
-        delegate :avro_schema, :value_avro_schema, :key_avro_schema, to: :config
+        delegate :avro_schema, :value_avro_schema, :key_avro_schema,
+                 :aliases, :inverse_aliases, to: :config
+
+        def inherited(subclass)
+          subclass.config = config
```

Comment:
I think it is `inherited` because the generated model class owns the configuration. If that model is subclassed then we want the subclass to refer back to the same configuration. 

I guess by definition that's a class variable, so we could switch to that for the config on a generated model class.